src/Foundation/Support/Configurations/CompileConfigurations.php

```diff
@@ -106,6 +106,7 @@ private function modal(Modal $component): array
             '5xl' => 'sm:max-w-5xl',
             '6xl' => 'sm:max-w-6xl',
             '7xl' => 'sm:max-w-7xl',
+            'full' => 'min-h-[96.8dvh] w-[100dvw]',
```

Comment:
The `full` idea is to open the modal in fullscreen, like `slide` component, and not with this blank space between.

![CleanShot 2024-09-24 at 11 44 00](https://github.com/user-attachments/assets/8bf7f298-2e0d-463b-aa5d-066e5c025f16)